source/_integrations/totalconnect.markdown

```diff
@@ -1,7 +1,6 @@
 ---
-title: Honeywell Total Connect Alarm
-description: Instructions on how to integrate TotalConnect alarms into Home Assistant.
-logo: honeywell-tc.png
+title: Total Connect
+description: Instructions on how to integrate Total Connect into Home Assistant.
```

Comment:
These changes are not related to this PR... However, that is not a really big problem.
The problem is that changing this title has no effect. It will be overwritten and automatically synced from the integration manifest:

<https://github.com/home-assistant/core/blob/dev/homeassistant/components/totalconnect/manifest.json#L3>

Please revert those 2 lines here.